cxx-sensors/src/main/java/org/sonar/cxx/sensors/utils/CxxIssuesReportSensor.java

```diff
@@ -267,12 +268,26 @@ private void saveViolation(SensorContext sensorContext, CxxReportIssue issue) {
       }
     }
 
+    for (CxxReportLocation location : issue.getFlow()) {
+      NewIssueLocation newIssueLocation = createNewIssueLocationFile(sensorContext, newIssue, location, affectedFiles);
+      if (newIssueLocation != null) {
+        newIssueFlow.add(newIssueLocation);
+      } else {
+        LOG.error("Failed to create new issue location from flow location {}", location);

```

Comment:
Should be debug not error.
Think makes no sense to mark whole analysis false in such a case.

Reply:
It should either be an error or a warning - otherwise such an error would go unnoticed.

Also the following clear() and break do not mark the whole analysis false - they just make sure we don't report an incomplete flow, which would be confusing. Other than that the issue remains perfectly valid.

Reply:
Hi, please have a look to https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Supported-configuration-properties#sonarcxxerrorrecoveryenabled how current error behaviour is. Realization is `in CxxUtils.validateRecovery(...)`.

Reply:
Hi @guwirth, I wasn't aware of this flag. I made some changes in error handling. Please have a look.